internal/sveltos/status.go

```diff
@@ -25,13 +25,14 @@ import (
 	hmc "github.com/Mirantis/hmc/api/v1alpha1"
 )
 
-// SetStatusConditions transforms status from Sveltos ClusterSummary
-// object and sets it into the provided list of conditions.
-func SetStatusConditions(summary *sveltosv1beta1.ClusterSummary, conditions *[]metav1.Condition) error {
+// GetStatusConditions returns a list of conditions from provided ClusterSummary.
+func GetStatusConditions(summary *sveltosv1beta1.ClusterSummary) ([]metav1.Condition, error) {
```

Comment:
Minor comment: I would keep an old `SetStatusConditions` name. Since the function still focuses on setting the conditions I think this name is still more appropriate than `GetStatusConditions`. What do you think?

It would also be helpful to mention in the comment above the function that it not only returns a list of conditions but also updates them.

Reply:
Yeah I think I agree. `GetStatusConditions` might suggest that we are retrieving status through an API call which is not the case.